url-ballerina/Package.md

```diff
@@ -1,5 +1,19 @@
 ## Package Overview
 
-This package provides the necessary utilities, which are required to encode and decode a URL or part of a URL.
+The URL library is one of the standard library modules of the [Ballerina](https://ballerina.io/) language.
 
-For information on the operations, which you can perform with this package, see the below **Functions**. For an example on the usage of the operations, see the [URL Encode/Decode Operations](https://ballerina.io/learn/by-example/url-encode-decode.html).
+URL encoding stands for encoding certain characters in a URL by replacing them with one or more character triplets that consist of the percent character "%" followed by two hexadecimal digits. The two hexadecimal digits of the triplet(s) represent the numeric value of the replaced character.
+
+The Ballerina URL module facilitates APIs to encode and decode a URL or part of a URL.
+
+For more information go to [The URL Module](https://docs.central.ballerina.io/ballerina/url/latest).
```

Comment:
shall we remove API Doc link from package.md. In central, there is a link to API Docs.

Reply:
Fixed with https://github.com/ballerina-platform/module-ballerina-url/pull/157/commits/bd4ac6a47962b94e53fc09a57ea157801831a4b4